configure.ac

```diff
@@ -251,6 +251,20 @@ AC_ARG_ENABLE(libscrypt,
 
 dnl --- Tracing Options. ---
 
+dnl USDT instrumentation option.
+AC_ARG_ENABLE(tracing-instrumentation-usdt,
+              AS_HELP_STRING([--enable-tracing-instrumentation-usdt],
+                             [build with tracing USDT instrumentation]))
```

Comment:
Somewhere, we should define what USDT is.

Reply:
Within the configure.ac? I thought of updating the `Tracing.md` file and it would explain all this.